bundleserver/src/main/java/net/discdd/server/BundleServerApplication.java

```diff
@@ -75,16 +75,18 @@ protected void refresh(ConfigurableApplicationContext applicationContext) {
             System.exit(1);
         }
 
-        app.setBanner((e, s, o) -> o.println(BananaUtils.bananansi("DDD Bundle Server", Font.ANSI_SHADOW, Ansi.GREEN)));
-
         if (CommandProcessor.checkForCommand(args)) {
             // we are doing a CLI command, so don't start up like a server
             app.setWebApplicationType(WebApplicationType.NONE);
             app.setBannerMode(Banner.Mode.OFF);
             app.setLogStartupInfo(false);
+            properties.put("logging.level.root", "error");
+        }
+        else {
+            app.setBanner((e, s, o) -> o.println(BananaUtils.bananansi("DDD Bundle Server", Font.ANSI_SHADOW, Ansi.GREEN)));
         }
         new BundleUploader().run(args);
-
+        app.setDefaultProperties(properties);
```

Comment:
we need this line before BundleUploader.run() as that was also using some server root from this properties file.

Or try running it once if it works like this in command mode too, it might've been changed since then.